docs/installation/pipx-based.rst

```diff
@@ -28,14 +28,14 @@ In the following sections, we will guide you step by step in configuring your sy
             python -m pipx ensurepath
             python -m pipx install muse-os
 
-    - After this, MUSE will be available to use system wide simply by invoking ``muse`` in the terminal, for example ``muse --model default``.
+    - After this, MUSE will be available to use system wide simply by invoking ``muse`` in the terminal. To illustrate this and to finish installation, run ``muse --model default``.
```

Comment:
Running the default model isn't really completing the installation. Remember, if users are using `pipx` they are just *running* MUSE, not *developing* it, so won't be running `pytest`.

Reply:
I know... I wanted to point out to a user to run this command as to stay out of trouble, should they become a developer and run the tests then. Or a potential developer reading this part of the documentation without running a default model.

Maybe I can just say something like:
- After this, MUSE will be available to use system wide simply by invoking ``muse`` in the terminal. To illustrate this and to test your installation, run ``muse --model default``. You should then see a a list of outputs printed to the terminal showing the computations going on in the background.

Alternatively, I can remove line 31 altogether and hope the respective users read the respective parts of the documentation as per intended use.

Reply:
This is much better. It's a good idea for users to check that things are working -- it was just the "finish the installation" bit that didn't sound quite right.